src/core/src/sketch/minhash.rs

```diff
@@ -622,12 +624,12 @@ impl KmerMinHash {
         self.check_compatible(other)?;
 
         let mut combined_mh = KmerMinHash::new(
-            self.num,
+            scaled_for_max_hash(self.max_hash),
```

Comment:
how about using
```suggestion
            self.scaled,
```
here? `self.scaled` is still using `max_hash` in its impl, but we can fix that later (and avoid a mention to `self.max_hash` here =])

Reply:
@luizirber sounds good. could you add more details about that pls? :)

Reply:
currently we have thousands of warnings in CI ([example](https://travis-ci.com/github/dib-lab/sourmash/jobs/376354282#L1605)) triggered because we are still using `.max_hash` internally. Eventually (for 4.0) we want to avoid using/documenting/suggesting `.max_hash` and focus on `.scaled` instead, including constructors (and hence why we deprecated `.max_hash` in 3.5 and generate all the warnings)

But that's all in the Python side, and this PR is for changing the Rust side. So, you don't need to worry about it if you don't want to, but I thought it was relevant to keep in mind =]

Reply:
I see .. thanks for the explanation :)